HanekeSwift.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
   s.name = 'HanekeSwift'
   s.module_name = 'Haneke'
-  s.version = '1.1'
+  s.version = '0.10.1.9'
```

Comment:
Why are you setting the version number to `0.10.1.9` and not `1.2`?

Reply:
this is the scheme we have been using. last was 0.10.1.8 so I just bumped the last number. not sure semantically what the components mean but since this change is backwards compatible it made sense.

Reply:
Echoing @j-j-m it looks like we point to this specific version in our Podfile:

```ruby
pod 'HanekeSwift', :git => 'git@github.com:stockx/HanekeSwift.git', :tag => 'v0.10.1.8'
```